cmd/migrations/2_currency.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/go-pg/migrations/v8"
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+	migrations.MustRegisterTx(func(db migrations.DB) error {
+		log.Println("Adding currency columns into invoices")
+		_, err := db.Exec(`
+			ALTER TABLE invoices 
+			ADD COLUMN amount_currency VARCHAR (5),
+			ADD COLUMN amount_without_vat_currency VARCHAR (5);
+			
+			update invoices set amount_currency = 'EUR';
+			update invoices set amount_without_vat_currency = 'EUR';
```

Comment:
let's capitalize keywords e.g. `UPDATE`

also we can add `UPDATE invoices SET amount_currency = 'SEK' WHERE amount = 12500`